core/jazz_ui/e2e/04-apiservices/apiservices.e2e-spec.ts

```diff
@@ -0,0 +1,316 @@
+/*
+ *Copyright 2016-2017 T Mobile, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); You may not use
+ * this file except in compliance with the License. A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or
+ * implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { browser, element, by, protractor, $ } from 'protractor';
+import { Jazz } from '../page-objects/jazzservices.po';
+import { CONFIGURATIONS } from '../../src/config/configuration';
+import { Timeouts, Browser } from 'selenium-webdriver';
+import { SSL_OP_SSLEAY_080_CLIENT_DH_BUG } from 'constants';
+
+const twok = 2000;
+const fivek = 5000;
+const tenk = 10000;
+const fifteenk = 15000;
+const twentyk = 20000;
+const thirtyk = 30000;
+const sixtyk = 60000;
+
```

Comment:
Use logical names and corresponding values.

Reply:
Will be better if we use a  shared config object, instead of hard coding in all files.